microservice/src/routes/helpers.rs

```diff
@@ -0,0 +1,53 @@
+use chrono::DateTime;
+use multiversx_sc_snippets::imports::RustBigUint;
+
+pub fn denominate(value: f64) -> u128 {
```

Comment:
small unit tests for these functions would be nice

Reply:
`denominate` function unit tests coming in PR #3 because it was refactored